lib/local-image.ts

```diff
@@ -76,6 +76,12 @@ export class LocalImage {
 	 */
 	public endTime?: number;
 
+	/**
+	 * This is the type of project that was resolved by this module, for
+	 * example Dockerfile.template or Dockerfile.arch.
+	 */
+	public projectType: string;
```

Comment:
Now I see why you've got the above, but the new SBVR we agreed yesterday allows projectType to be null.

Reply:
Ah gotcha, yeah I prefer that.